plexpy/webserve.py

```diff
@@ -242,6 +245,7 @@ def libraries(self):
         return serve_template(templatename="libraries.html", title="Libraries", config=config)
 
     @cherrypy.expose
+    @addtoapi()
     def get_library_list(self, **kwargs):
```

Comment:
All the datatable calls won't work with the API without providing the `json_data`.